src/components/card/CardSidebar.vue

```diff
@@ -205,6 +205,10 @@ export default {
 				padding-top: $modal-padding;
 				z-index: 100;
 				background-color: var(--color-main-background);
+
+				.app-sidebar-header__menu {
+					top: -7px;
```

Comment:
I think it would be more reasonable to just remove the padding-top here so that the header is aligned at the very top again.

Reply:
Basically getting rid of https://github.com/nextcloud/deck/blob/master/src/components/card/CardSidebar.vue#L205 should do the trick here.

Reply:
Ok, I removed `padding-top`.
Speaking about [this](https://github.com/nextcloud/deck/blob/master/src/components/card/CardSidebar.vue#L205), I tried to remove it too, but it doesn't change anything.

Reply:
The actual modal width is defined here https://github.com/nextcloud/deck/blob/master/src/App.vue#L152-L156, the rule you linked is just to workaround as we misuse the sidebar component here a bit :wink: